src/main/validators/formValidator.ts

```diff
@@ -45,7 +46,27 @@ export class FormValidator {
         if (errors && errors[0] && errors[0].message) {
           const message = `${key}.${errors[0].message}`;
           item._errors.push(message);
-          validationErrors[item.id] = message;
+          if (item._flagComment) {
+            validationErrors[`flagComment-${item.id}`] = message;
+          } else if (
+            item._listOfValuesLength > 0 &&
+            item._listOfValuesLength > JSON.parse(config.get('radio.listOfValuesLength')) &&
```

Comment:
why don't you extract `JSON.parse(config.get('radio.listOfValuesLength'))` into a local variable? as you do this 6 times..

Reply:
I tried this but that was giving for config to allow mutation .

Reply:
What do you mean by that? couldn't you do something like `const listOfValuesLength = JSON.parse(config.get('radio.listOfValuesLength'))`

Reply:
worked now